tools/pika_cdc/README.md

```
@@ -0,0 +1,24 @@
+# Pika cdc
+**A tool for incremental synchronization of pika command**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)